### PR TITLE
fix(GLCanvasElement): GLCanvasElement not working on Uno Islands + automatic framebuffer resizing

### PR DESCRIPTION
**Pull request by ramezgerges**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/AddIns/Uno.WinUI.Graphics3DGL/Uno.WinUI.Graphics3DGL.csproj [hunk 1]
```diff
@@ -66,17 +66,23 @@
...
 		<DownloadFile
+			Condition="!exists('$(MSBuildThisFileDirectory)angle_binaries/osx/libGLESv2.dylib')"
 			SourceUrl="https://github.com/dotnet/Silk.NET/raw/a9c75a61528ee7b4e55442f5141cb132f10d72ff/src/Native/Silk.NET.OpenGLES.ANGLE.Native/runtimes/osx/native/libGLESv2.dylib"
 			DestinationFolder="$(MSBuildThisFileDirectory)angle_binaries/osx">
 		</DownloadFile>
 	</Target>
 
+	<Target Name="CleanAngle" AfterTargets="Clean">
+		<RemoveDir Directories="$(MSBuildThisFileDirectory)angle_binaries" />
+	</Target>
```
Comment (Youssef1313) [inline src/AddIns/Uno.WinUI.Graphics3DGL/Uno.WinUI.Graphics3DGL.csproj:84]: That's likely not the right way. The right way is to do `<FileWrites Include="angle_binaries/**" />` in an `ItemGroup` just after the download task.

Comment (jeromelaban) [reply src/AddIns/Uno.WinUI.Graphics3DGL/Uno.WinUI.Graphics3DGL.csproj:84]: Let's fix that in a separate PR.

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.Desktop.targets [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	-->
 	<Import Project="$(MSBuildThisFileDirectory)Uno.Implicit.Packages.ProjectSystem.Uno.targets" />
 
-	<ItemGroup Condition="$(UnoFeatures.Contains(';canvas3dgl;'))">
+	<ItemGroup Condition="$(UnoFeatures.Contains(';GLCanvas;'))">
```
Comment (Youssef1313) [inline src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.Desktop.targets:9]: This should be `glcanvas` (lowercase) I think we we normalize UnoFeatures to lower case

File: src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.WinAppSdk.targets [hunk 3]
```diff
@@ -14,7 +14,7 @@
 		<_UnoProjectSystemPackageReference Include="SkiaSharp.Views.WinUI" ProjectSystem="true" />
 	</ItemGroup>
 
-	<ItemGroup Condition="$(UnoFeatures.Contains(';canvas3dgl;'))">
+	<ItemGroup Condition="$(UnoFeatures.Contains(';GLCanvas;'))">
```
Comment (Youssef1313) [inline src/Uno.Sdk/targets/Uno.Implicit.Packages.ProjectSystem.WinAppSdk.targets:17]: This should be `glcanvas` (lowercase) I think we we normalize UnoFeatures to lower case

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.5

**Pull request merged**